### ISSUE TITLE
Stashed tabs are not appearing in sidebar

### ISSUE THREAD

**Issue by stephen-petrick** [josh-berry/tab-stash#113]
### System Details

- **OS Version:** [Windows 10 Home Version 10.0.19042 Build 19042]
- **Browser Version:** [Firefox 85.0.2]
- **Tab Stash Version:** [2.8]

**Installed Browser Extensions:**

- Facebook container (issue has occurred both with and without the container enabled)
- Adblock Plus
- Avira Browser Safety
- Default Bookmark folder
- DuckDuck Go Privacy Essentials
- EPUBReader
- HTTPS Everywhere
- LastPass: Free Password Manager
- MOBI Reader
- Simple Translate
- Video DownloadHelper

### Describe the Bug

When attempting to stash one or more tabs, the stashed tab(s) would disappear from Firefox but were unable to be unstashed via sidebar buttons.
A new group would be created in the sidebar, but the new group would be empty despite having just stashed a tab.

### Expected Behavior

A new group in the sidebar would be created containing and listing the stashed tab(s). Upon clicking one of the "open tab(s)" buttons, all selected stashed tabs would be restored and accessed through the browser.

### Steps to Reproduce

Unfortunately I'm not sure how to recreate the problem from a previously functioning state.
Most I can currently provide is 
1. Open a new tab and access a website (ex: wikipedia.org)
2a. Stash the tab via right clicking the tab, choosing Tab Stash > Stash this tab
2b. Alternatively, stash the tab via sidebar buttons, clicking "Stash only the unstashed tabs to a new group" or "Stash this tab"
3. Observe a new group is created in the sidebar with current date, but stashed tab is not listed underneath the new group
![image](https://user-images.githubusercontent.com/36829101/108641637-e35d6600-7465-11eb-879f-b5a1e6f1ee30.png)

### Impact

Unable to use Tab Stash to create new groups of stashed tabs. 

I have tried uninstalling and reinstalling Tab Stash
I have tried disabling and re-enabling Facebook Container
I have restarted my browser.

### Additional Details

I've observed several additional behaviors that I do not believe Tab Stash or (possibly) Firefox had performed before.

One is that instead of storing tabs in "Other bookmarks > Tab Stash", stashed tabs are instead placed under Bookmark Menu.
Two is that stashed tabs are instead stored under "Hidden Tabs". Firefox gives a popup claiming Tab Stash is causing this behavior.
Example of popup from Firefox:
![image](https://user-images.githubusercontent.com/36829101/108641699-4cdd7480-7466-11eb-8ed2-67c0b4befc6f.png)

Three is that some stashed tabs are recorded in Bookmark Menu as "about:blank#stashing-R_WCb_eyM7Rc-0" or "about:blank#stashing-8df0xw-G01RV-0", with the characters after "stashing-" varying for each tab

However, I am still able to restore older groups of stashed tabs.

<hr>
<em>Readers: If you are also experiencing this bug, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the issue if you have more details to add, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Hi, sorry to hear you're having trouble! One extension in your list potentially stands out to me as causing conflicts: "Default bookmark folder". Have you installed, updated or changed settings on this extension recently? From a cursory inspection, it seems "Default bookmark folder" will move ANY newly-created bookmark into a designated location (whether or not it was created via the extension).

If that is indeed what it's doing, this could cause the problem you're seeing in Tab Stash. For performance reasons, Tab Stash will create these temporary `about:blank#stashing...` bookmarks in parallel, and then update them in parallel with the actual URLs/titles of your stashed tabs. (This is faster than creating the bookmarks one by one, and ensures they appear in the correct order once all is said and done.) If the "Default bookmark folder" extension is moving these temporary bookmarks before Tab Stash can update them, it's quite possible that the stashing will fail. Or, if the bookmarks are moved AFTER Tab Stash has updated them, Tab Stash will think the stashing has succeeded and close your tabs even though the bookmarks have been moved out of position.

Can you please try disabling "Default bookmark folder" and see if this resolves your issue? Or, check and see if there is an option not to move ALL newly-created bookmarks?

Regarding the hidden-tab popup, this is expected—by default, Tab Stash will not close your stashed tabs, merely hide them (so they will appear in the drop-down menu but not in the tab bar itself). Tab Stash will manage these hidden tabs in the background so you don't have to worry about memory usage. If you truly want these tabs to be closed instead of hidden, you can control this behavior through the Tab Stash options.

Sorry for the inconvenience and I hope this helps!

**Comment by stephen-petrick** [comment]
Ah, that was indeed it. Thank you very much!

Changing the "Select where new bookmarks will be added" setting to either "Last folder used" or "no folder selected" from "Bookmark menu" seemed to have solved the problem.

Thanks again!

**Comment by josh-berry** [comment]
Glad to hear it! I'm going to close this out but feel free to open another issue if you run into any more trouble.
